pkg/nuclide/ui/tree/lib/TreeRootComponent.js

```diff
@@ -178,6 +190,35 @@ var TreeRootComponent = React.createClass({
     this.setState({expandedKeys});
   },
 
+  async _toggleChildNodesExpanded(parentNode: LazyTreeNode, forceExpanded: ?boolean): Promise {
+    if(isPathIgnored(parentNode.getItem().getPath())) {
+      return;
+    };
+    var childNodes = parentNode.getCachedChildren();
+
+    // fetch children if cache is empty
+    if(!childNodes) {
+      await parentNode.fetchChildren().then(nodes => childNodes = nodes);
```

Comment:
Avoid single-line arrow functions if the function body is an assignment because single line arrows implicitly return the value of the expression.

```js
await parentNode.fetchChildren().then(nodes => {
  childNodes = nodes;
});
```